counter-sink/src/test/java/org/springframework/cloud/stream/module/metrics/CounterSinkSimpleNameTests.java

```diff
@@ -62,10 +63,11 @@ public void clear() {
     }
 
     @Test
-    public void testIncrement() {
+    public void testIncrement() throws InterruptedException {
         assertNotNull(this.sink.input());
         Message<String> message = MessageBuilder.withPayload("Hi").build();
         sink.input().send(message);
+        Thread.sleep(5500);
```

Comment:
We obviously don't want this.
Need to discuss the approach. This PR uses asynchronous offloading of values to redis as opposed to directly writing to redis. This test may be refactored to exercise the in-memory store instead. cc @markpollack